docs/sql-reference/sql-functions/array-functions/array_contains_seq.md

```diff
@@ -0,0 +1,46 @@
+# array_contains_seq
+
+## Description
+
+Checks whether all the elements of array2 appear in array1 in the same exact order. Therefore, the function will return 1, if and only if array1 = prefix + array2 + suffix.
+
+## Syntax
+
+~~~Haskell
+BOOLEAN array_contains_all(arr1, arr2)
+~~~
+
+## Parameters
+
+`arr`: the two arrays to compare. This syntax checks whether `arr2` is a subset of `arr1` and in the same exact order.
+
+The data types of elements in the two arrays must be the same. For the data types of array elements supported by StarRocks, see [ARRAY](../../../sql-reference/sql-statements/data-types/Array.md).
+
+## Return value
+
+Returns a value of the BOOLEAN type.
+
+1 is returned if `arr2` is a subset of `arr1`. Otherwise, 0 is returned.
+
+## Examples
```

Comment:
append more special msg like in https://clickhouse.com/docs/en/sql-reference/functions/array-functions#hassubstr, such as null, no supertype etc.

Reply:
we prefer keeping the same behavior with trino